model/registry/db.yaml

```
@@ -203,30 +203,25 @@ groups:
           in order to map the path part values to their names.
         examples: ['db.elasticsearch.path_parts.index=test-index', 'db.elasticsearch.path_parts.doc_id=123']
         tag: tech-specific-elasticsearch
-      - id: mssql.instance_name
-        type: string
-        stability: experimental
-        note: >
-          If setting a `db.mssql.instance_name`, `server.port` is no longer
-          required (but still recommended if non-standard).
-        brief: >
-          The Microsoft SQL Server [instance name](https://docs.microsoft.com/sql/connect/jdbc/building-the-connection-url?view=sql-server-ver15)
-          connecting to. This name is used to determine the port of a named instance.
-        examples: 'MSSQLSERVER'
-        tag: tech-specific-mssql
-      - id: name
+      - id: collection.namespace
         type: string
         stability: experimental
         brief: >
-          This attribute is used to report the name of the database being accessed.
-          For commands that switch the database, this should be set to the target database
-          (even if the command fails).
-        note: >
-          In some SQL databases, the database name to be used is called "schema name".
-          In case there are multiple layers that could be considered for database name
-          (e.g. Oracle instance name and schema name),
-          the database name to be used is the more specific layer (e.g. Oracle schema name).
-        examples: [ 'customers', 'main' ]
+          The namespace containing database objects, fully qualified within the server address and port.
+        note: |
+          For many database systems, the namespace matches the database (catalog, keyspace, schema) name.
+          In general case, however, namespace consist of several qualifiers:
+
+          * Instance name - MS SQL Server or Oracle, allow to host multiple database engines on the same host.
+          * Database name - The name of the database, keyspace, namespace, etc.
+          * Schema name - such as PostgreSQL or MS SQL Server schemas.
+          * Other sub-namespaces which may include partition or shard identifiers.
+
+          Namespace SHOULD include all such identifiers applicable to the database system and available to the instrumentation.
+
+          Semantic conventions for individual database systems SHOULD document what `db.collection.namespace`
+          means in the context of that system.
```

my thought is to try to stick to general statements here and then lean on

> Semantic conventions for individual database systems SHOULD document what `db.collection.namespace` means in the context of that system.

```suggestion
          If a database system has multiple namespace components, they should be concatenated (potentially using database system specific conventions) from most general to most specific namespace component, and more specific namespaces should not be captured without the more general namespaces, to ensure that "startswith" queries for the more general namespaces will be valid.

          Semantic conventions for individual database systems SHOULD document what `db.collection.namespace`
          means in the context of that system.
```